src/draco/draco_features.h

```diff
@@ -0,0 +1,15 @@
+// GENERATED FILE -- DO NOT EDIT
```

Comment:
Can you add more details to the commit message on how this file was generated so it can be regenerated next upgrade?

Reply:
Yep. Would a `draco_update.md` file in docs be a better place to put it?

Reply:
I would opt for both actually. One in the commit history so it can be found there and the same in the doc that will contain more of the data. This can be added if you're just going to squash all these commits too so we can have a single commit to carry around.

Reply:
Sounds good, I'll add it to the commit message and the doc then